### ISSUE TITLE
Shortcut for 'Format SQL' not working

### ISSUE THREAD

**Issue by alchemyy** [dbeaver/dbeaver#15415]
#### System information: 
- Windows10
- DBeaver 21.3.4
- No Additional extensions

#### Describe the problem you're observing: 
- Ctrl+Shift+F not working, while 'Format SQL' under 'Edit' menu works fine.

**Comment by uslss** [comment]
@alchemyy Probably, you've got Format Active Query Only option enabled. Could you try to disable it and check if it helps?
![image](https://user-images.githubusercontent.com/31996417/152945186-0c295070-cbf1-49e2-84aa-2898d035dc30.png)

**Comment by alchemyy** [comment]
Hey @uslss , Thanks for the reply! I just figured that out , the DBeaver shortcut has collision to my system's shortcut for INPUT METHOD, which changes my Chinese input from 'Simplified Chinese' to 'Traditional Chinese'. Hard to figure that out though. ,would be better if DBeaver had the hotkey collision check integrated. 

P.S. Found a tool to check hotkey usage--OpenARK, however it did not show the hotkey mentioned above, and I haven't figured this out...maybe it just doesn't cover all the hotkey system-wide.
